### ISSUE TITLE
USER ISSUE: give rep kick froms erver

### ISSUE THREAD

**Issue by Sheo42** [StrangeLoopGames/EcoIssues#24715]
\
**User Id: 94.101**
**Version:** 0.10.1.3 beta release-592

**Steps to Reproduce:**
1. give rep to a picture

**Expected behavior:**
Rep is given to the picture and I can keep playing :)

**Actual behavior:**
The following error message popped up:
```
Server encountered an exception:

<size=60,00%>Exception: NullReferenceException
Message:Object reference not set to an instance of an object.
Source:Eco.Gameplay

System.NullReferenceException: Object reference not set to an instance of an object.
   at Eco.Gameplay.Civics.Laws.LegalActions.Money.TaxPayBase_LegalAction.GovernmentTransferUsers(LocString description, Single amount, Currency currency, BankAccount governmentAccount, GameValue`1 aliasGameValue, IContextObject context, Settlement jurisdictionSettlement, Boolean canTransferNonCitizens, Boolean taxing, Boolean preventIfUnableToPay, AccountChangeSet acc)
   at Eco.Gameplay.Civics.Laws.LegalActions.Money.Pay_LegalAction.DoTransfer(LocString description, IContextObject context, Settlement jurisdictionSettlement, Single amount, Currency currency, Boolean preventIfUnableToPa</size>...

Check log file to see full message.
```

**Do you have mods installed? Does the issue happen when no mods are installed?:**
No mod, plublic server Giant Panda

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by D3nnis3n** [comment]
Which picture is that exactly, in which town is it and which town are you a citizen of?

**Comment by Sheo42** [comment]
Town is Ocean view
Picture is one of those : 
![20240201195028_1](https://github.com/StrangeLoopGames/EcoIssues/assets/88681087/1c8b1de8-3a13-4ac2-816e-d71fce351b09)

Sorry i don't remember which one. 
Also, I am not a member of the twon and not a citizen

Looking at the logs, it might have something to do with the fact that in my town (Redwood Forest) we get paid for giving reputation to the pictures.

**Comment by D3nnis3n** [comment]
This should be fixed in 10.2, reopen if you ever see it again.
